src/shared/Atlassian.Bitbucket/BitbucketHostProvider.cs

```diff
@@ -10,6 +10,8 @@ namespace Atlassian.Bitbucket
 {
     public class BitbucketHostProvider : IHostProvider
     {
+        public const bool USE_BEARER_TOKEN = true;
```

Comment:
Rather than introduce these constants, I think it'd be better to use `true` and `false` at the call sites, but use a named parameter. For example:

```csharp
_bitbucketApi.GetUserInformationAsync(null, accessToken, isBearerToken: true)
```